src/components/Thanks/ThanksPageSingleVersion.vue

```diff
@@ -145,11 +158,34 @@ const scrollToReceipt = () => {
 	setTimeout(() => {
 		receiptSection.value?.orderConfirmationContainer?.scrollIntoView({ behavior: 'smooth' });
 	}, 500);
+
+	$kvTrackEvent(
+		POST_CHECKOUT_EVENT_CATEGORY,
+		CLICK_EVENT_ACTION,
+		'open-order-confirmation-drawer',
+		userType.value,
+		numberOfBadges,
+	);
+};
+
+const scrollToAccountCreation = () => {
```

Comment:
Do we need to scroll to account creation section or open the modal?

Reply:
In consistency having the account creation section below, scrolling makes sense to me, if product has a feedback about this we can fix it...

Reply:
The creation modal is from the MyKiva TY page, so I followed that existing pattern.

Reply:
Said that, would created a ticket to add modal to this new page?

Reply:
It's already there, you should be able to use the `handleContinue` method in `ThanksPageSingleVersion`

Reply:
Oh nice, thanks!

Reply:
I locally merged that work after mine 😆

Reply:
Oh no! I tried to get it in fast enough 😿